includes/data-events/connectors/class-connector.php

```diff
@@ -97,7 +97,7 @@ public static function order_completed( $timestamp, $data, $client_id ) {
 			return;
 		}
 
-		static::put( $contact );
+		static::put( $contact, 'RAS Order completed' );
```

Comment:
```suggestion
		static::put( $contact, sprintf( 'RAS Order completed. Order ID: %d', $order_id ) );
```

Reply:
I just merged one similar suggestion but something occurred me.

One thing this makes it more difficult is to filter the logs by context. Let me see all the log entries for "RAS Order completed" for a certain site... for example. The user ID should be present already in the `data`. Same thing for the previous and new status of the subscriptions I just approved...